internal/services/topic.go

```diff
@@ -163,8 +162,15 @@ func (ts *TopicService) Reconcile() (TopicReconcileResult, error) {
 		} else {
 			result.Assignments = ts.currentAssignments(topicMetadata)
 		}
-
+	} else {
+		labels := prometheus.Labels{
+			"topic": ts.canaryConfig.Topic,
+		}
+		describeTopicError.With(labels).Inc()
+		glog.Errorf("Error retrieving metadata for topic %s: %v", ts.canaryConfig.Topic, topicMetadata.Err.Error())
```

Comment:
`Error()` provides a formatted string so the `%s` instead of `%v` is enough.

Reply:
Changed the `%v` to `%s`.